BenchmarkTests/README.md

```diff
@@ -0,0 +1,83 @@
+# Benchmark Tests
+
+[Synthetics for Mobile](https://docs.datadoghq.com/mobile_app_testing/) runs Benchmark test scenarios to collect metrics of the SDK performances.
+
+
+## CI
+
+CI continuously builds, signs, and uploads a runner application to Synthetics which runs predefined tests daily.
+
+### Build
+
+Before building the application, the `BenchmarkTests/xcconfigs/Benchmark.local.xcconfig` configuration file must be present and contain the `Mobile - Integration Org` client token and RUM application ID, and API Key. These values are sensitive and must be securely stored.
+
+```ini
+CLIENT_TOKEN=
+RUM_APPLICATION_ID=
+API_KEY=
+```
+
+### Sign
+
+To sign the runner application, the certificate and provision profile defined in [Synthetics.xcconfig](xcconfigs/Synthetics.xcconfig) and in [exportOptions.plist](exportOptions.plist) needs to be installed on the build machine. These files are sensitive and must be securely stored. Make sure to update both files when updating the certificate and provisioning profile, otherwise signing fails.
+
+> [!NOTE]
+> Certificate & Provisioning Profile are also available through the [App Store Connect API](https://developer.apple.com/documentation/appstoreconnectapi). But we don't have the tooling in place.
+
+### Upload
+
+The application version (build number) is set to the commit SHA of the current job, and the build is uploaded to Synthetics using the [datadog-ci](https://github.com/DataDog/datadog-ci) CLI. This step expects environment variables to authenticate with the `Mobile - Integration Org`:
+
+```bash
+export DATADOG_API_KEY=
+export DATADOG_APP_KEY=
+export S8S_APPLICATION_ID=
+```
+
+## Development
+
+Each scenario is independent and can be considered as an app within the runner.
+
+### Create a scenario
+
+A scenario must comply with the [`Scenario`](Runner/Scenarios/Scenario.swift) protocol. Upon start, a scenario initializes the SDK, enables features, and returns a root view-controller.
+
+Here is a simple example of a scenario using Logs:
+```swift
+import Foundation
+import UIKit
+
+import DatadogCore
+import DatadogLogs
+
+struct SessionReplayWebViewScenario: Scenario {
```

Comment:
suggestion/ The name of example scenario seems wrong. In general, I'd avoid providing examples in `README.md` for things that may change rapidly in next iterations making  snippets obsolete. Better to reference some existing scenario (e.g. `/Runner/Scenarios/SessionReplay/SessionReplayScenario.swift`).

In practice, when adding new scenarios we will unlikely refer to this `README.md` but rather clone & adjust some of existing ones.
```suggestion
struct LogsScenario: Scenario {
```

Reply:
I still think it's valuable to show a simple exemple in the readme. That's the one that should be copied, instead of a real one that will have a more complex setup.